Xwt.Gtk/Xwt.GtkBackend/PopoverBackend.cs

```diff
@@ -262,10 +276,10 @@ public void Show (Xwt.Popover.Position orientation, Xwt.Widget reference, Xwt.Re
 
 			popover.Hidden += (o, args) => sink.OnClosed ();
 
-			var screenBounds = reference.ScreenBounds;
+			var screenBounds = reference.Allocation;
```

Comment:
Why is this change needed? `Widget.Allocation` is not reliable, especially for custom drawn widgets and widgets without own `Gdk.Window` and/or Gtk widgets inside an environment mixed with Cocoa. This is the reason why I used `ScreenBounds` here. Additionally the coords are used to `Move` the Popover to the destination screen coordinates, while `Allocation` is relative to the parent or window in most cases.